maquetacion/scss/tech.scss

```diff
@@ -0,0 +1,41 @@
+.tech-box-card {
+  align-items: center;
+  background-color: $green;
+  border-radius: $tech-radius;
+  display: flex;
+  flex-direction: column;
+  height: $tech-height;
+  justify-content: space-around;
+  margin: $margin-tech 10px;
```

Comment:
Try to use variables for values that would be repeated on other elements, if it will be used only on this element is better just to use the number